File: Models/Address.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;

namespace FinancialPortal.Models
{
    public class Address
    {
        public long AddressId { get; set; }

        public string StreetAddress { get; set; }

        public string City { get; set; }

        public string State { get; set; }

        public string PostalCode { get; set; }

        public long? CustomerId { get; set; }// Foreign key
        public Customer Customer { get; set; } // Reference navigation
    }
}

File: ClientApp/src/app/portal/Models/Address.ts
export class Address {
  addressID: number;
  streetAddress: string;
  city: string;
  state: string;
  postalCode: string;
  constructor() {
    this.addressID = 0;
    this.streetAddress = '';
    this.city = '';
    this.state = '';
    this.postalCode = '';
  }
}
